project_pivot/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Projects Pivot View",
```

Comment:
Please use proper English here. This is not a technical name, where we omit prepositions, articles, etc.

```suggestion
    "name": "Pivot view for projects",
```